### DISCUSSION TITLE
How to use convention symbol like mkdocs for material site?

### DISCUSSION THREAD

**Discussion by Aqours** [squidfunk/mkdocs-material#6020] [Q&A] [answered]
<img width="314" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/8586286/f92349be-4bb7-4e02-b264-9313c64fbbb9">

I add `<!-- md:version 9.2.0 -->` to md file, it not works.
Where are some docs?

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking. This is implemented as a [custom hook](https://github.com/squidfunk/mkdocs-material/blob/master/src/.overrides/hooks/shortcodes.py) and some [custom CSS](https://github.com/squidfunk/mkdocs-material/blob/f9d5c6709b2692cf083e413c9d944dc9d59b4079/src/.overrides/assets/stylesheets/custom/_typeset.scss#L96-L132). I'm not sure if this is generalizable enough to be useful, but you can certainly copy our sources and even adapt the shortcodes if necessary ☺️

**Reply by hellt** [reply]
this is a very cool feature for insiders, I'd love to have it (and, frankly, I feel that `version` is a must have for any sw project using mkdocs-material for their docs).

**Reply by danielloader** [reply]
Another vote for genericising this, it's an excellent way to avoid using Mike - just having version conventional tags in paragraph headings is a fantastic way to do it. Not that this is news... given you did it and it's worked well.

**Comment by hellt** [comment]
For future explorers, I used mkdocs-macros-plugin and used CSS styles from the accepted answer to achieve the same effect - https://github.com/srl-labs/learn-srlinux/commit/405068a787b5ff6a6b0c95779587c8750cc4d924
